python/cudf/cudf/core/index.py

```diff
@@ -2490,6 +2495,57 @@ def round(self, freq):
 
         return self.__class__._from_data({self.name: out_column})
 
+    def tz_localize(self, tz, ambiguous="NaT", nonexistent="NaT"):
+        """
+        Localize timezone-naive data to timezone-aware data.
+
+        Parameters
+        ----------
+        tz: str
```

Comment:
I think there's usually supposed to be a space on both sides of the colon here.

> The colon must be preceded by a space, or omitted if the type is absent.

https://numpydoc.readthedocs.io/en/latest/format.html#parameters

```suggestion
        tz : str
```

Reply:
Fix'd